src/PSDMatrices.jl

```diff
@@ -33,29 +33,29 @@ end
 # Custom functions
 
 function todense(M::PSDMatrix)
-    return M.L * M.L'
+    return M.R' * M.R
 end
 
 function X_A_Xt(A::PSDMatrix, X::AbstractMatrix)
-    return PSDMatrix(X * A.L)
+    return PSDMatrix(A.R * X')
 end
 
 function add_cholesky(A::PSDMatrix, B::PSDMatrix)
     sum_dense = todense(A) + todense(B)
-    factor = cholesky(sum_dense).L
+    factor = cholesky(sum_dense).U
     return PSDMatrix(factor)
 end
 
 function add_qr(A::PSDMatrix, B::PSDMatrix)
-    stack = hcat(A.L, B.L)
+    stack = vcat(A.R, B.R)
     matrix = PSDMatrix(stack)
     return choleskify_factor(matrix)
 end
 
 function choleskify_factor(M::PSDMatrix)
-    R = qr(M.L').R
+    R = qr(M.R).R
     R = nonnegative_diagonal(R)
-    return PSDMatrix(LowerTriangular(R'))
+    return PSDMatrix(UpperTriangular(R))
```

Comment:
essentially, this is the kind of improvement going from L to R. It is admittedly small, but in my view, the rest of the code is equivalent, so I might prefer this way. WDYT?

Reply:
I have to admit that I'm slightly afraid of the consequences of this choice, since I'm very used to having it the other way round :sweat_smile: But since QR is such an integral part of this idea, I agree that it makes sense to think in terms of R^T R :+1:

Reply:
Same. But it might be the fountain of youth to have these kinds of changes every now and then ;)